features/step_definitions/generate.rb

```diff
@@ -72,12 +72,28 @@
     derived_data_path: @derived_data_path
   )
   flunk("Product with name #{product} and destination #{destination} not found in DerivedData") if info_plist_path.nil?
-
+  
   unless system("/usr/libexec/PlistBuddy -c \"print :#{key}\" #{info_plist_path}")
     flunk("Key #{key} not found in the #{product} Info.plist")
   end
 end
 
+Then("the product {string} with destination {string} contains the Info.plist key {string} with value {string}") do |product, destination, key, value|
```

Comment:
beautiful 👏 thanks for doing this

Reply:
This was something new for me and I really liked it, haha 👍